### PR TITLE
feat(CTA-29) Get User

### PR DESCRIPTION
**Pull request by ebond07**
JIRA: https://champlainsaintlambert.atlassian.net/jira/software/projects/CTA/boards/59?selectedIssue=CTA-29
## Context:
Adding the ability for an admin to get a specific user's details
## Changes
- Added backend methods required to retrieve a single client's details through their clientId
- Added backend testing for said methods
- Modified frontend of the accounts page to allow for a dropdown to show the client's details
- Added frontend testing for the additional elements added to the accounts page
- Added a footer
- Modified the styling for the navbar
### Before
image
### After
image
image
## Dev notes (Optional)
- Frontend in the docker-compose.yml is commented out so we can bring the frontend up separately to insure auto refreshing

### CODE REVIEW

File: coupetonarbrefrontend/src/components/AccountsAdmin.jsx [hunk 1]
```diff
@@ -21,42 +23,51 @@ function AccountsAdmin() {
...
-  
+
   useEffect(() => {
-    const fetchClients = () => {
-      axios.get('http://localhost:8080/users/clients')
+    if (selectedClientId !== null) {
+      axios.get(`http://localhost:8080/users/clients/${selectedClientId}`)
         .then(response => {
-          setClients(response.data);
+          setSelectedClientDetails(response.data);
```
Comment (Matthewtan9) [inline coupetonarbrefrontend/src/components/AccountsAdmin.jsx:55]: Why did you change the setClients to setSelectedClientDetails.

**Review by Matthewtan9 — APPROVED** [review]
Good job.

File: coupetonarbrefrontend/src/components/AccountsAdmin.jsx (hunk 1 above)
Comment (ebond07) [reply coupetonarbrefrontend/src/components/AccountsAdmin.jsx:55]: Thought it made it more clear, instead of setting any client, it is taking the specific client details of the client that you've clicked on.

Comment (Matthewtan9) [reply coupetonarbrefrontend/src/components/AccountsAdmin.jsx:55]: Thank you for your response, Mr. Bond. Very nice.

**Review by dazhuox — APPROVED** [review]
Great work, it's good that you created a footer component on top of your story, it made the pages one step closer to completion.
